must_triage/inspectors/ocp.py

```diff
@@ -72,3 +73,27 @@ def pod_ready(obj):
                 f"Container '{cs['name']}' in "
                 f"pod '{obj['metadata']['name']}' is not ready")
         return result
+
+    @staticmethod
+    def restart_count(obj):
+        result = list()
+        status = obj['status']
+        try:
+            container_statuses = status['containerStatuses']
+        except KeyError:
```

Comment:
You wanted us to update the code on this
> If 'containerStatuses' is present in status, and skip the check if it is not.

Does it mean that we should check for the condition 'containerStatuses' is present in status first and if it is present then it should perform the restart_count function and if not the exit from that?